website/docs/r/variable_set.html.markdown

```diff
@@ -126,6 +126,64 @@ resource "tfe_variable" "test-b" {
 }
 ```
 
```

Comment:
Not sure if adding both makes the example section too long, but I thought it would be valuable to make it clear that a project-owned varset doesn't necessarily/automatically apply itself to the project and that it can be scoped to workspaces.

Reply:
Definitely, great idea to make that behavior clear.